CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Dangerzone 0.4.1 (unreleased)
+
+- Platform support: Drop Fedora 35, since it's end-of-life ([issue #308](https://github.com/freedomofpress/dangerzone/issues/308))
+- Feature: Introduce PySide6 / Qt6 support on Windows, MacOS, and Linux (dev-only) ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Feature: Adjust conversion timeouts based on the document's pages/size, and
+  allow users to disable them ([issue #327](https://github.com/freedomofpress/dangerzone/issues/327))
+- Feature: Convert documents with non-standard permissions or SELinux labels ([issue #335](https://github.com/freedomofpress/dangerzone/issues/335))
```

Comment:
#335 is the PR that specifically closes this, but maybe we should mention the parent issue instead: https://github.com/freedomofpress/dangerzone/issues/157

Reply:
The PR that closed this is https://github.com/freedomofpress/dangerzone/pull/336a actually. I used #335 here because it's the umbrella issue for these kinds of problems (file accessible via group, SELinux issues). Same as https://github.com/freedomofpress/dangerzone/pull/366#discussion_r1129312598 though, if you want to I can mention all the affected issues.